packages/app/src/app/pages/Dashboard/Components/SyncedSandbox/SyncedSandboxListItem.tsx

```diff
@@ -19,6 +20,7 @@ export const SyncedSandboxListItem = ({
   onDoubleClick,
   onContextMenu,
   isScrolling,
+  isViewOnly,
```

Comment:
same question as above, should we even consider the synced sandboxes part of the limitation?

Reply:
I don't know actually. When implementing this it felt weird to me that we wouldn't add the view only or restrictions to synced sandboxes, so I just did it for these as well.